### DISCUSSION TITLE
How to update

### DISCUSSION THREAD

**Discussion by MrFacundo** [vuejs/vitepress#2268] [Q&A] [answered]
I am trying to update the vitepress version

`// package.json

{
	"scripts": {
		"docs:dev": "vitepress dev files",
		"docs:build": "vitepress build files",
		"docs:serve": "vitepress serve files"
	},
	"devDependencies": {
		"vitepress": "^0.22.4"
	}
}
`
`❯ npm list vitepress
└── vitepress@0.22.4`

I've tried 

npm update vitepress
npm install -D vitepress
npm update
npm update -D

But the version in package.json is still the same

How can I update it?

**Comment by brc-dd** [comment, marked as answer]
npm update does not work like that. You either need to manually specify the version, something like `"vitepress": "latest"` or use something like npm-check-updates or taze.

**Comment by Zhengqbbb** [comment]
get a try ～ https://github.com/antfu/taze ！

```
npx taze
```
